src/Mod/TechDraw/TechDrawTools/CommandCopyView.py

```diff
@@ -0,0 +1,99 @@
+# ***************************************************************************
+# *                                                                         *
+# *   Copyright (c) 2022 - Wanderer Fan <wandererfan@gmail.com>             *
```

Comment:
Please remove line 2 (line 3 should be line 2)  
The format is `Copyright (c) 2022 Wanderer Fan <wandererfan@gmail.com>` (no dash necessary)